ci-operator/config/jaegertracing/jaeger-operator/jaegertracing-jaeger-operator-main.yaml

```diff
@@ -0,0 +1,98 @@
+base_images:
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+  operator-sdk:
+    name: "4.14"
+    namespace: origin
+    tag: operator-sdk
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+binary_build_commands: LD_FLAGS="-X github.com/jaegertracing/jaeger-operator/pkg/version.version=1.46.0
+  -X github.com/jaegertracing/jaeger-operator/pkg/version.buildDate=2023-06-28T14:39:00Z
+  -X github.com/jaegertracing/jaeger-operator/pkg/version.defaultJaeger=1.46.0" make
+  build
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.14
+images:
+- build_args:
+  - name: VERSION
+    value: 1.46.0
+  - name: JAEGER_VERSION
+    value: 1.46.0
+  - name: VERSION_DATE
+    value: "2023-06-28T14:39:00Z"
+  - name: VERSION_PKG
+    value: github.com/jaegertracing/jaeger-operator/pkg/version
+  context_dir: .
+  dockerfile_path: Dockerfile
+  to: jaeger-operator
+- context_dir: .
+  dockerfile_path: Dockerfile.asserts
+  to: jaeger-tests-asserts-e2e
+- context_dir: .
+  dockerfile_path: tests/Dockerfile
+  to: jaeger-tests-runner
+operator:
+  bundles:
+  - as: jaeger-bundle
+    context_dir: .
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: quay.io/jaegertracing/jaeger-operator:1.46.0
+    with: pipeline:jaeger-operator
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
```

Comment:
```suggestion
      version: "4.14"
```

Reply:
For presubmits, we have kept the version to 4.13 as there are issues with claiming 4.14 clusters.